### PR TITLE
Re-ran mstar and reaction example notebooks; updated version mentions in readme

### LINKED ISSUE #989: Documentation not quite correct re python version
We actually need python 2.7.8 as the minimum version, I happen to have access to a particularly decrepid 2.7.5 distro which fails to get cram stuff.

### PR DESCRIPTION
**Pull request by ZoeRichter**
I updated the readme where it mentioned python 2.7 dependency/use and version of ubuntu before 16.04.  However, I have not touched any of the sections on Mac.

I also ran the m* and reaction example notebooks, which weren't broken, but needed to be run again to get rid of the QA warnings.

### CODE REVIEW

**Comment by ZoeRichter** [comment]
I genuinely don't understand how my changes could make a py3 test fail.

**Review by opotowsky — COMMENTED** [review]
This looks good!

I think you just need to force push the newly atomic mass updates up to your branch so CI can rerun.

**Comment by ZoeRichter** [comment]
Wait, I see now.  The data test isn't updated to reflect the new data library values.

**Comment by opotowsky** [comment]
> Wait, I see now. The data test isn't updated to reflect the new data library values.

Let me know if you need any help with the git stuff.

**Comment by bam241** [comment]
This need to be rebase against the `0.7-rc` branch I think...

**Comment by ZoeRichter** [comment]
How do I rebase, exactly?

**Comment by ZoeRichter** [comment]
this branch was made off of 0.7.0

**Review by ngranda — APPROVED** [review]
This looks good to me.

File: readme.rst [hunk 1]
```diff
@@ -94,16 +94,16 @@ or from public sources on the internet.
...
-on Python 2.7
+on Python 3.5+
 
- * Ubuntu 12.04 - x86_64
+ * Ubuntu 18.04 - x86_64
  * OSX 10.8 - x86_64
 
-PyNE has pre-built binaries for the lastest release (0.4) on windows
-mac and linux 
+PyNE has pre-built binaries for the lastest release (0.7) on windows
```
Comment (kkiesling) [inline readme.rst:102]:
```suggestion
PyNE has pre-built binaries for the lastest release (0.7) on 
```
removing windows reference

Comment (kkiesling) [reply readme.rst:102]: not sure if this suggestion overlaps with @nsryan2 #1285 where he changes the readme.

**Review by kkiesling — CHANGES_REQUESTED** [review]

**Review by kkiesling — APPROVED** [review]

**Pull request merged**